src/IJ_mv/protos.h

```diff
@@ -189,9 +191,12 @@ HYPRE_Int HYPRE_IJMatrixSetDiagOffdSizes ( HYPRE_IJMatrix matrix, const HYPRE_In
 HYPRE_Int HYPRE_IJMatrixSetMaxOffProcElmts ( HYPRE_IJMatrix matrix, HYPRE_Int max_off_proc_elmts );
 HYPRE_Int HYPRE_IJMatrixRead ( const char *filename, MPI_Comm comm, HYPRE_Int type,
                                HYPRE_IJMatrix *matrix_ptr );
+HYPRE_Int HYPRE_IJMatrixReadBinary ( const char *filename, MPI_Comm comm, HYPRE_Int type,
```

Comment:
Why is `HYPRE_IJMatrixReadBinary` not in `HYPRE_IJ_mv.h`?

Reply:
Nice catch! I forgot to put it there... Fixed with [478857a](https://github.com/hypre-space/hypre/pull/826/commits/478857a9411676cbb407a53e0ed07f1234783ee7)